build/import/Packages.targets

```diff
@@ -93,8 +94,8 @@
     <PackageReference Update="Roslyn.Diagnostics.Analyzers"                                           Version="3.3.0" />
 
     <!-- NuGet -->
-    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="5.8.0-preview.2.6776" />
-    <PackageReference Update="NuGet.VisualStudio"                                                     Version="6.0.0-preview.0.2" />
+    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="6.0.0-preview.1.89" />
```

Comment:
Given that VS as a whole stopped using interops, NuGet followed suit and merged NuGet.VisualStudio and NuGet.SolutionRestoreManager.Interop into 1 package. 

The currently inserted version in `6.0.0-preview.2.129`. 

I'd expect no compilation/functional issues.

Reply:
It seems that this package version is not in the package sources in Package.Targets.

Which package source do you use for 6.0.0-preview2.129 ?

Reply:
The same package source should work.

Replace NuGet.SolutionRestoreManager.Interop with `NuGet.VisualStudio`.

Reply:
I did this change in another PR https://github.com/dotnet/project-system/pull/7398